kouigenjimonogatari/.htaccess

```diff
@@ -1,6 +1,11 @@
 Options +FollowSymLinks
 RewriteEngine on
 
+RewriteRule ^/imc/(.*) http://diyhistory.org/tmp/convert/index.php/$1 [R=302,L]
+RewriteRule ^imc2/(.*) http://diyhistory.org/tmp/convert/index.php/$1 [R=302,L]
+RewriteRule ^imc3(.*) http://diyhistory.org/tmp/convert/index.php/$1 [R=302,L]
+RewriteRule ^/imc4(.*) http://diyhistory.org/tmp/convert/index.php/$1 [R=302,L]
```

Comment:
Is it intentional that each of these has a different pattern?  If this is just testing rules, perhaps we could help?

Reply:
This is a testing rule.
I appreciate your help.

I want to do the following redirect.

http://w3id.org/kouigenjimonogatari/imc/http://example.org
=> http://diyhistory.org/tmp/convert/index.php/http://example.org

Reply:
Use this form:
```apache
RewriteRule ^imc/(.*) http://diyhistory.org/tmp/convert/index.php/$1 [R=302,L]
```
Based on local tests, it will do this:
```
curl -v https://w3id.org/kouigenjimonogatari/imc/http://example.org
...
> GET /kouigenjimonogatari/imc/http://example.org HTTP/1.1
...
< HTTP/1.1 302 Found
< Location: http://diyhistory.org/tmp/convert/index.php/http:/example.org
...
```

Reply:
Thank you so much for your help.
I will update.